whitepaper/README.md

```diff
@@ -8,9 +8,6 @@ Bruce Fenton<br>
 Tron Black<br>
 http://www.ravencoin.org<br>
 3rd April 2018
-</p>
-<p align="center"><i>
-In the fictional world of Westeros, ravens are used as messengers who carry statements of truth. Ravencoin is a use-case focused blockchain designed to carry statements of truth about who owns what assets.
 </i></p> 
```

Comment:
this closing `</i>` would need to be removed as well

Reply:
thanks for pointing that out

Reply:
no problem!